packages/authorization/src/authorize-interceptor.ts

```diff
@@ -55,19 +68,27 @@ export class AuthorizationInterceptor implements Provider<Interceptor> {
       resource: invocationCtx.targetName,
       invocationContext: invocationCtx,
     };
+
     debug('Security context for %s', description, authorizationCtx);
     let authorizers = await loadAuthorizers(
       invocationCtx,
       metadata.voters || [],
     );
+
+    // pass `currentUser` to authorizers
+    Object.assign(metadata, {currentUser: user});
```

Comment:
`currentUser` should be set to `authorizationContext` if necessary instead of `metadata`.

Reply:
https://github.com/strongloop/loopback-next/pull/3529 is also updated accordingly.